### DISCUSSION TITLE
smooth scrolling

### DISCUSSION THREAD

**Discussion by Dickby** [qutebrowser/qutebrowser#7279] [Q&A]
Hi,
today it happened that for a while commands like crtl-d and ctrl-u  made the website scroll smooth.
This never was the case on my system and i didn't knew about it. It stopped working after closing the browser though.
I have scrolling.smooth enabled, but it only works for 'j' and 'k' as it was before.
How can i enable that again? It was beautiful!

**Comment by The-Compiler** [comment]
No clue I'm afraid - smooth scrolling is entirely handled by the underlying QtWebEngine/Chromium, all qutebrowser does is [enabling it](https://github.com/qutebrowser/qutebrowser/blob/1996c4d7e72bb3764cd4ea316f8cf84d4fc32dca/qutebrowser/browser/webengine/webenginesettings.py#L144-L145).
